modules/phase_field/doc/content/documentation/systems/VectorPostprocessors/FeatureVolumeVectorPostprocessor.md

```diff
@@ -17,11 +17,17 @@ When using this VPP with a FeatureFloodCount object, there is no implied orderin
 - var_num - The index of which coupled variable represents this feature [0..n).
 - feature_volume - The volume of the feature, computed as an integral of the solution over each element representing this feature.
 - intersects_bounds - A Boolean indicating whether this feature intersects any boundary.
+- centroid_<x, y, z> (optional) - The coordinates of each feature centroid (Currently only supported when no periodic boundaries are used).
 
 ## Typical Output
 
 !listing grain_tracker_volume_out_grain_volumes_0000.csv
 
+## Centroid Output
+
+This VectorPosptrocessor can also output centroid information when the simulation is not using periodic boundary conditions. To enable
```

Comment:
> when the simulation is not using periodic boundary conditions

It would be nice if this was enforced somewhere.

Reply:
It kind-of is. I could change this to an error:
https://github.com/idaholab/moose/blob/devel/modules/phase_field/src/postprocessors/FeatureFloodCount.C#L704-L708

Reply:
Or we could just figure out how to calculate centroids properly for arbitrary periodic BCs.